Gem/Code/Source/Components/Multiplayer/JumpPadComponent.cpp

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project. For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzCore/Component/ComponentApplicationBus.h>
+#include <AzCore/Component/TransformBus.h>
+#include <AzFramework/Physics/PhysicsSystem.h>
+#include <Source/Components/Multiplayer/JumpPadComponent.h>
+
+#include "PlayerJumpPadEffectComponent.h"
+
+namespace MultiplayerSample
+{
+    void JumpPadComponent::Reflect(AZ::ReflectContext* context)
+    {
+        AZ::SerializeContext* serializeContext = azrtti_cast<AZ::SerializeContext*>(context);
+        if (serializeContext)
+        {
+            serializeContext->Class<JumpPadComponent, JumpPadComponentBase>()
+                ->Version(1);
+        }
+        JumpPadComponentBase::Reflect(context);
+    }
+
+    void JumpPadComponent::GetRequiredServices(AZ::ComponentDescriptor::DependencyArrayType& required)
+    {
+        JumpPadComponentBase::GetRequiredServices(required);
+        required.push_back(AZ_CRC_CE("PhysicsTriggerService"));
+    }
+
+    void JumpPadComponent::OnInit()
+    {
+    }
+
+    void JumpPadComponent::OnActivate([[maybe_unused]] Multiplayer::EntityIsMigrating entityIsMigrating)
+    {
+    }
+
+    void JumpPadComponent::OnDeactivate([[maybe_unused]] Multiplayer::EntityIsMigrating entityIsMigrating)
+    {
+    }
+
+
+    JumpPadComponentController::JumpPadComponentController(JumpPadComponent& parent)
+        : JumpPadComponentControllerBase(parent)
+    {
+    }
+
+    void JumpPadComponentController::OnActivate([[maybe_unused]] Multiplayer::EntityIsMigrating entityIsMigrating)
```

Comment:
Do you have a controller even on this entity on the client machine? It should be in NetEntityRole client, and not have any controller at all..